.github/workflows/sync-release-to-gitcode.yml

```
@@ -2,7 +2,7 @@ name: Sync Release to GitCode
 
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)